modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -97,4 +97,124 @@ Status description:: error: connection exception - alias chains are not permitte
 
 === 08N15
 
-Status description:: error: connection exception - no such routing policy. Policy definition of the routing policy { $routingPolicy } could not be found. Verify that the spelling is correct.
\ No newline at end of file
+Status description:: error: connection exception - no such routing policy. Policy definition of the routing policy { $routingPolicy } could not be found. Verify that the spelling is correct.
+
+[[_data-exceptions]]
+== Data exceptions
+
+The following is a list of the data exceptions that can occur in Neo4j.
+
+=== 22N00 
+
+Status description:: error: data exception - unsupported value. The provided value is unsupported and cannot be processed.
+
+=== 22N01 
+
+Status description:: error: data exception - invalid type.
+Expected the value { $value } to be of type { $valueTypeOredList }, but was of type { $valueType }.
```

Comment:
The 'Ored' or 'Anded' in some list parameters is mostly a way to show which format it should be written out on. Made up example to show the principle:

The element(s) { $valueTypeList } are broken => "The element(s) a, b**,** c are broken"
The element(s) { $valueTypeOredList } are broken => "The element(s) a, b **or** c are broken"
The element(s) { $valueTypeAndedList } are broken => "The element(s) a, b **and** c are broken"

In the codebase we just call the key "valueTypeList" in all cases and have a separate internal joinstyle thing which tracks which of these three formats should be used. I'm not sure whether it is helpful or just confusing for the user to call it  `{ $valueTypeOredList }` or if we should simply call it `{ $valueTypeList }` like in the code? 

When we add examples later those will show on which format we print out the list in. When we reintroduce the statusParameter in the diagnostic record in a later release (it was descoped for now) that will be a map with just the key and the list as a Java list without any formatting, so there we probably want to call it valueTypeList.
 
 Any thoughts in this @NataliaIvakina and @renetapopova as it applies to a bunch of other places as well.

Reply:
IMO, `{ $valueTypeList }` is less confusing

Reply:
I agree with Natalia. I have already created a card to our wall to add the parameter descriptions, which I think will eliminate any confusion. How did you come up with 'Ored' and 'Anded' instead of just { $valueTypeOrList } and { $valueTypeAndList }?

Reply:
Ok let's go for { $valueTypeList } then. This also applies to a few other codes in this PR with list parameters, I think it was like 3 or so more occurrences . 

That is a good question, I think that was Stefan's idea when he did re-designed/formalized how parameter framework works. I guess he is used to writing GQL standard because that whole writing is a bit over-complicated... And then Isak probably used his wording when implementing it.